webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMSSmartling.java

```diff
@@ -968,6 +977,12 @@ private TextUnitSearcherParameters baseParams(
     result.setPluralFormsExcluded(pluralFormsExcluded);
     result.setSkipTextUnitWithPattern(skipTextUnitsWithPattern);
     result.setSkipAssetPathWithPattern(skipAssetsWithPathPattern);
+    result.setExcludeUnexpiredPendingMT(
+        aiTranslationConfiguration != null && aiTranslationConfiguration.isEnabled());
+    result.setAiTranslationExpiryDuration(
+        aiTranslationConfiguration != null
+            ? aiTranslationConfiguration.getExpiryDuration()
+            : Duration.ofHours(3));
```

Comment:
It seems like we shouldn't need to have this default in here at all:
`Duration.ofHours(3)`

We either have a valid aiTranslationConfiguration - that includes an expiry duration with that config - or, if we absolutely need a default when that is missing (vs. erroring out saying that the config is invalid) - to have that pulled to a more visible location vs. it being defined deep in the code.